deposit/hal/metadata.py

```diff
@@ -148,7 +148,10 @@ def render(self, paper, filename, form):
         language.attrib['ident'] = 'en'  # TODO adapt this?
         textClass = addChild(profileDesc, 'textClass')
 
-        domains = [form.cleaned_data['topic'].lower()]
+        domains = []
+        if 'cleaned_data' in dir(form):
```

Comment:
In this case "form" is None.

This fix is a workaround, I don't know the root cause.

However, it's a must-have to proceed.